client/app/scripts/liveblog-edit/timeline.js

```diff
@@ -10,6 +11,45 @@ define([
         var blog = {
             _id: $route.current.params._id
         };
+
+        $scope.$on('posts', function() {
+            var filter = [
+                {range: {
+                    _updated: {
+                            gte: $scope.lastUpdated
+                        }
+                    }
+                }
+            ];
+            return api('blogs/<regex(\"[a-f0-9]{24}\"):blog_id>/posts', blog).query({
+                source: {
+                    query: {filtered: {filter: {
+                        and: filter
+                    }}},
+                    sort: [{versioncreated: 'asc'}]
+                }
+            }).then(function(data) {
+                $scope.lastUpdated = moment.utc().format();
```

Comment:
Yes, so if your time is set off it can be set off aswell, but then this is your fault :),
Anyway like I sayed this is a temp solution the proper solution is with cid.